src/server/list_family.cc

```diff
@@ -1290,7 +1310,8 @@ void ListFamily::Register(CommandRegistry* registry) {
       << CI{"LREM", CO::WRITE, 4, 1, 1, acl::kLRem}.HFUNC(LRem)
       << CI{"LMOVE", CO::WRITE | CO::NO_AUTOJOURNAL, 5, 1, 2, acl::kLMove}.HFUNC(LMove)
       << CI{"BLMOVE", CO::WRITE | CO::NO_AUTOJOURNAL | CO::BLOCKING, 6, 1, 2, acl::kBLMove}
-             .SetHandler(BLMove);
+             .SetHandler(BLMove)
+      << CI{"LMPOP", CO::WRITE | CO::FAST, -4, 1, -1, acl::kLMPop}.HFUNC(LMPop);  // Added LMPOP command
```

Comment:
I dont see kLMPop is defined

Reply:
how did you compile and run the unit tests?

Reply:
Thanks for your review, @adiholden

You're right—I missed defining kLMPop. I'll add that in the correct place.

To be completely honest, I made the code changes but haven't compiled or run the unit tests yet. I'm not familiar with the compilation process for this project. Could you guide me on how to compile the project and run the tests so I can properly verify my changes before submitting?

I’ll update the PR once I’ve made the necessary changes and successfully compiled everything.